frontend/src/components/account/AccountPage.js

```diff
@@ -193,6 +208,91 @@ export default function AccountPage({
     </Link>
   );
 
+  // Lines 210 - 300 handle follow button for organizations
```

Comment:
Comments referencing line numbers are not very future proof.

Reply:
bump :smile:

Reply:
Removed line referencing